code/game/machinery/doors/firedoor.dm

```diff
@@ -299,7 +302,7 @@
 		return
 	..()
 	latetoggle()
-	layer = 2.6
+	layer = base_layer

```

Comment:
This is going to end up with firedoors over airlocks when both are open, leading to people not able to close the airlock how they want.